### ISSUE TITLE
how to add a lot contacts

### ISSUE THREAD

**Issue by rohalah** [vysheng/tg#566]
hi
how way to add a lot contacts.
i tried with add_contact but not add after 100 - 200 contact
please help me
thanks

**Comment by LukeLR** [comment]
Maybe you should Split the List of contacts in several batches of around 100 contacts (or what amount works well) and then add 100 contacts, quit tg-cli, start it again (maybe use the clean command to clean it's offline files if needed) and then add the next batch of 100 contacts, maybe this solves the problem...

**Comment by rohalah** [comment]
thanks
but I tried to add contact and limit by 450 and don't add any contact.
I tried to add contact in windows app but don't add any contact. 
Are limit by telegram.

**Comment by LukeLR** [comment]
Yes, you're right, If you can't add more contacts in other Telegram applications as well, this seems to be a telegram-related bug (or even a limit)? It seems strange that telegram limits the Number of contacts, so I think this might be a bug in your account as well (even if this seems equally unlikely). You should contact the telegram support (in Telegram go to Settings — Ask a question) or add +42449 to your contacts.

**Comment by pazis** [comment]
Same problem here. Can't add more than 450 contacts. 😢

**Comment by lauriolasw** [comment]
Hello
I have a similar problem: I can only add a few contacts (5-10) and then, telegram-cli always says:
"[]" (json void)

Even if the number is correctly registered  on Telegram.
After that, if I use the official Telegram App for Android, the result is the same:
"is not on Telegram yet, would you like to invite them to join?"

Why can I only add few contacts?
Thanks
